### PR TITLE
chore: Add node auth tests to CircleCI

### PR DESCRIPTION
**Pull request by amhinson**
_Description of changes:_
- Run Node.js integ tests for `amazon-cognito-identity-js` in CircleCI

Successful CircleCI run: https://app.circleci.com/pipelines/github/aws-amplify/amplify-js/7362/workflows/b6ca2c1d-742b-44c1-b691-e333c0fd34b4/jobs/34966

Integ sample & tests: https://github.com/aws-amplify/amplify-js-samples-staging/pull/206

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -111,6 +111,26 @@ commands:
...
+        type: string
+      category:
+        type: string
+      sample_name:
+        type: string
+    steps:
+      - run:
+          name: 'Install << parameters.test_name >> sample'
+          command: |
+            echo "Current NPM registry: " $(yarn config get registry)
```
Comment (sammartinez) [inline .circleci/config.yml:126]: Whats the reason for the echo?

Comment (amhinson) [reply .circleci/config.yml:126]: Not sure honestly. That's just what all the other commands do 🙂

**Review by iartemiev — APPROVED** [review]
LGTM!

File: .circleci/config.yml (hunk 1 above)
Comment (iartemiev) [reply .circleci/config.yml:126]: It's there just to confirm we're publishing to Verdaccio instead of NPM proper

Comment (iartemiev) [reply .circleci/config.yml:126]: Not actionable at that point, but just for the sake of having explicit logs

**Pull request merged**